dojo/db_migrations/0072_sheets_textfields.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 2.2.17 on 2021-01-30 07:23
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('dojo', '0071_product_type_enhancement'),
```

Comment:
we already passed this number.
```suggestion
        ('dojo', '0073_product_type_enhancement'),
```
Also rename the file